Gordon360/Controllers/ProfilesController.cs

```diff
@@ -93,12 +93,19 @@ public ProfilesController(IProfileService profileService, IAccountService accoun
             {
                 student = _student == null ? null : (PublicStudentProfileViewModel)_student;
                 faculty = _faculty == null ? null : (PublicFacultyStaffProfileViewModel)_faculty;
-                alumni = null;  //student can't see alumini
+                // If this student is also in Alumni AuthGroup, then s/he can see alumni's public profile; if not, return null.
+                alumni = _alumni == null ? null : viewerGroups.Contains(AuthGroup.Alumni) ? (PublicAlumniProfileViewModel)_alumni : null;
+            }
+            else if (viewerGroups.Contains(AuthGroup.Alumni))
+            {
+                student = null;
+                faculty = _faculty == null ? null : (PublicFacultyStaffProfileViewModel)_faculty;
+                alumni = _alumni == null ? null : (PublicAlumniProfileViewModel)_alumni;
             }
 
             if (student is null && alumni is null && faculty is null)
             {
-                return Ok(null);
+                return Ok();
```

Comment:
Why did you remove the `null` here?

Reply:
You should test this via Swagger and see if the response is the same or not.

Reply:
Will do!

Reply:
I checked with Swagger. They show differently but I'm not sure if they mean the same thing. It's hard to describe the differences, so I changed it back to the original state and maybe we can discuss which one is better later. Thanks!